build-logic/build-update-utils/src/main/kotlin/gradlebuild/buildutils/tasks/UpdateInitPluginTemplateVersionFile.kt

```diff
@@ -50,7 +50,7 @@ abstract class UpdateInitPluginTemplateVersionFile : DefaultTask() {
     abstract val libraryVersionFile: RegularFileProperty
 
     @TaskAction
-    fun updateInitPluginTemplateVersionFile() {
+    fun action() {
```

Comment:
detekt - MemberNameEqualsClassName: A member is named after the class. This might result in confusion. Either rename the member or change it to a constructor.

Reply:
I think having a descriptive name for `@TaskAction` has value. Can this check be disabled for methods annotated with `@TaskAction`?

Reply:
I neither agree nor have any good counter arguments. I can't configure the rule the way you suggested (based on the `@TaskAction` annotation), but I can disable it, because, frankly I don't really see the value it's adding. Will do that.